tpkg/concurrent_credns.tdir/concurrent_credns.test

```diff
@@ -0,0 +1,68 @@
+# #-- concurrent_credns.test --#
+# source the master var file when it's there
+[ -f ../.tpkg.var.master ] && source ../.tpkg.var.master
+# use .tpkg.var.test for in test variable passing
+[ -f .tpkg.var.test ] && source .tpkg.var.test
+
+. ../common.sh
+
+export NSD_PATH="../../nsd"
+
+echo "** Starting servers"
+make start-slave
+wait_nsd_up slave/nsd.log
+make start-proxy
+wait_nsd_up proxy/nsd.log
+make start-master
+wait_nsd_up proxy/nsd.log
```

Comment:
The last one is supposed to be wait_nsd_up for master/nsd.log, instead of proxy/nsd.log, I think?

Reply:
Would make sense...

Reply:
Done